src/catch.h

```diff
@@ -0,0 +1,12 @@
+// Minetest
```

Comment:
Please explain/document that this header should be included instead of <catch.hpp> because it patches the output streams to rawstream.

Reply:
done